components/oneapi-validation/src/main/java/com/wso2telco/dep/oneapivalidation/service/impl/location/ValidateLocation.java

```diff
@@ -128,4 +129,37 @@ public void validateUrl(String pathInfo) throws CustomException {
 
         UrlValidator.validateRequest(requestParts, validationRules);
     }
+    
+    /**
+     * Returns array of MSISDNs
+     */
+    public String[] getMsisdns() {
+		return this.addresses.toArray(new String[addresses.size()]);
+	}
+
+    /**
+     * Returns array of MSISDNs without "tel:+" prefix
+     */
+	public String[] getUserMsisdns() {
```

Comment:
Since the purpose of the oneapi-validation is validate the properties, it would be better if you could move this method into mediation-dep

Reply:
Moved to utill class ValidationUtils within mediation-dep